### ISSUE TITLE
reg_loss always 1.0 on custom dataset

### ISSUE THREAD

**Issue by brucetusec** [tianzhi0549/FCOS#310]
I tried training this model on my own dataset,

cls_loss converges fine, but reg_loss always 1.0, and pred_left, pred_right, pred_top, pred_bottom become all zero after 5~10 iteration.

Would you please give me some suggestions?

**Comment by justustulo** [comment]
@brucetusec 
I know I'm kind of late, but I'm having the same issue as well with the reg_loss stuck around 1 on VOC. Still don't have a solution, were you able to figure it out?

**Comment by brucetusec** [comment]
use torch.exp for regression output works in my case. I am using linear IOU loss now.
